docs/src/compatibility.md

```diff
@@ -0,0 +1,69 @@
+# **6.** Compatibility
+
+Compatibility refers to the ability to restrict what version of the dependencies that your project is compatible with.
+If the compatibility for a dependency is not given, the project is assumed to be compatible with all versions of that dependency.
+
+Compatibility for a dependency is entered in the `Project.toml` file as for example:
+
+```toml
+[compat]
+julia = "1.0"
+Example = "0.4.3"
+```
+
+After a compatibility entry is put into the project file, `up` can be used to apply it.
+
+The format of the version specifier is described in detail below.
+
+!!! info
+    There is currently no way to give compatibility from the Pkg REPL mode so for now, one has to manually edit the project file.
+
+## Version specifier format
+
+Similar to other package managers, the Julia package manager respects [semantic versioning](https://semver.org/) (semver).
+As an example, a version specifier is given as e.g. `1.2.3` is therefore assumed to be compatible with the versions `[1.2.3 - 2.0.0)` where `)` is a non-inclusive upper bound.
+More specifically, a version specifier is either given as a **caret specifier**, e.g. `^1.2.3`  or a **tilde specifier** `~1.2.3`.
+Caret specifiers are the default and hence `1.2.3 == ^1.2.3`. The difference between a caret and tilde is described in the next section.
+The intersection of multiple version specifiers can be formed by comma separating indiviual version specifiers.
```

Comment:
Do you mean `union` rather than `intersection`? For example if I want to support julia 0.7 and 1.0 I can write 
```julia
 [compat]
julia = "0.7, 1.0"
```
I think this is worth emphasizing with an example, given the semver rules specially about version 0 (making the caret etc less useful) and the fact that so many packages currently use 0.x versions.

Also `JuliaRegistries/General` uses a dash syntax, is that a special case or valid here too?